auth-service/app/src/api/v1/auth.py

```diff
@@ -49,7 +51,6 @@ async def register(
 async def login(
```

Comment:
логин не работает, падает с ошибкой

```
2023-09-22 16:02:59 2023-09-22 11:02:59,768 INFO sqlalchemy.engine.Engine BEGIN (implicit)
2023-09-22 16:02:59 2023-09-22 11:02:59,769 INFO sqlalchemy.engine.Engine SELECT "user".id, "user".login, "user".password, "user".first_name, "user".last_name, "user".email, "user".created_at 
2023-09-22 16:02:59 FROM "user" 
2023-09-22 16:02:59 WHERE "user".email = %s
2023-09-22 16:02:59 2023-09-22 11:02:59,769 INFO sqlalchemy.engine.Engine [cached since 25.58s ago] ('user@example.com',)
2023-09-22 16:02:59 2023-09-22 11:02:59,778 INFO sqlalchemy.engine.Engine SELECT user_1.id AS user_1_id, role.id AS role_id, role.name AS role_name, role.description AS role_description 
2023-09-22 16:02:59 FROM "user" AS user_1 JOIN userroles AS userroles_1 ON user_1.id = userroles_1.user_id JOIN role ON role.id = userroles_1.role_id 
2023-09-22 16:02:59 WHERE user_1.id IN (%s)
2023-09-22 16:02:59 2023-09-22 11:02:59,779 INFO sqlalchemy.engine.Engine [generated in 0.00065s] (UUID('5fcbba62-ac6d-4731-b400-85d2887fb3b9'),)
2023-09-22 16:02:59 2023-09-22 11:02:59,784 INFO sqlalchemy.engine.Engine SELECT "user".password 
2023-09-22 16:02:59 FROM "user" 
2023-09-22 16:02:59 WHERE "user".email = %s
2023-09-22 16:02:59 2023-09-22 11:02:59,784 INFO sqlalchemy.engine.Engine [generated in 0.00013s] ('user@example.com',)
2023-09-22 16:02:59 2023-09-22 11:02:59,947 INFO sqlalchemy.engine.Engine ROLLBACK
2023-09-22 16:02:59 [2023-09-22 11:02:59 +0000] [27] [ERROR] Exception in ASGI application
2023-09-22 16:02:59 Traceback (most recent call last):
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 408, in run_asgi
2023-09-22 16:02:59     result = await app(  # type: ignore[func-returns-value]
2023-09-22 16:02:59              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
2023-09-22 16:02:59     return await self.app(scope, receive, send)
2023-09-22 16:02:59            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/fastapi/applications.py", line 292, in __call__
2023-09-22 16:02:59     await super().__call__(scope, receive, send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/applications.py", line 122, in __call__
2023-09-22 16:02:59     await self.middleware_stack(scope, receive, send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 184, in __call__
2023-09-22 16:02:59     raise exc
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 162, in __call__
2023-09-22 16:02:59     await self.app(scope, receive, _send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 79, in __call__
2023-09-22 16:02:59     raise exc
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
2023-09-22 16:02:59     await self.app(scope, receive, sender)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
2023-09-22 16:02:59     raise e
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
2023-09-22 16:02:59     await self.app(scope, receive, send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 718, in __call__
2023-09-22 16:02:59     await route.handle(scope, receive, send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 276, in handle
2023-09-22 16:02:59     await self.app(scope, receive, send)
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 66, in app
2023-09-22 16:02:59     response = await func(request)
2023-09-22 16:02:59                ^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 273, in app
2023-09-22 16:02:59     raw_response = await run_endpoint_function(
2023-09-22 16:02:59                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 190, in run_endpoint_function
2023-09-22 16:02:59     return await dependant.call(**values)
2023-09-22 16:02:59            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/home/src/api/v1/auth.py", line 68, in login
2023-09-22 16:02:59     access_token = await auth_service.create_access_token(
2023-09-22 16:02:59                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/home/src/services/auth.py", line 79, in create_access_token
2023-09-22 16:02:59     access_token = await self.auth.create_access_token(
2023-09-22 16:02:59                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/async_fastapi_jwt_auth/auth_jwt.py", line 277, in create_access_token
2023-09-22 16:02:59     return await self._create_token(
2023-09-22 16:02:59            ^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/async_fastapi_jwt_auth/auth_jwt.py", line 196, in _create_token
2023-09-22 16:02:59     return jwt.encode(
2023-09-22 16:02:59            ^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/jwt/api_jwt.py", line 67, in encode
2023-09-22 16:02:59     json_payload = self._encode_payload(
2023-09-22 16:02:59                    ^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/site-packages/jwt/api_jwt.py", line 94, in _encode_payload
2023-09-22 16:02:59     return json.dumps(
2023-09-22 16:02:59            ^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/json/__init__.py", line 238, in dumps
2023-09-22 16:02:59     **kw).encode(obj)
2023-09-22 16:02:59           ^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/json/encoder.py", line 200, in encode
2023-09-22 16:02:59     chunks = self.iterencode(o, _one_shot=True)
2023-09-22 16:02:59              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/json/encoder.py", line 258, in iterencode
2023-09-22 16:02:59     return _iterencode(o, 0)
2023-09-22 16:02:59            ^^^^^^^^^^^^^^^^^
2023-09-22 16:02:59   File "/usr/local/lib/python3.11/json/encoder.py", line 180, in default
2023-09-22 16:02:59     raise TypeError(f'Object of type {o.__class__.__name__} '
2023-09-22 16:02:59 TypeError: Object of type Role is not JSON serializable
```

Reply:
починил